stdlib/in_channel.mli

```diff
@@ -51,6 +51,19 @@ val open_gen : open_flag list -> int -> string -> t
     file permissions.  {!open_text} and {!open_bin} are special cases of this
     function. *)
 
+val with_open_bin : string -> (t -> 'a) -> 'a
+(** [with_open_bin fn f] opens file [fn] for reading, and passes the resulting
+    channel to [f]. The channel is closed before [f] returns with a value or by
```

Comment:
Actually the channel is not closed before `f` returns, but after :-) 

I would formulate this doc string differently. Here's a tentative:

```
[with_open_bin fn f] opens a channel [ic] on file [fn] and returns [f ic]. After 
the function returns, either with a value or by raising an exception, [ic] is 
guaranteed to be closed.
```

Reply:
Thanks, fixed as per suggestion.